source/llvm/CodeGenBase.h

```diff
@@ -40,7 +43,7 @@ class CodeGenBase
 
         if(functionPassManager)
         {
-            functionPassManager->run(*func);
+            //functionPassManager->run(*func);
```

Comment:
If this call indeed turned out to not be needed, let's remove the entire if{} statement.

Reply:
I'm not yet sure whether this is needed or not. We already have module level optimizations, so do we also need function level optimizations? Also, what impact do they actually make? More passes requires more build time, but for how much gain? I left this commented out as a reminder to do some profiling with and without.